<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>显著性分析</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>327</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>组别</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_group">
      <item>
       <property name="text">
        <string>VWD类型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>血友病类型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出血/血栓</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>血型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>性别</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_data">
      <item>
       <property name="text">
        <string>年龄</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>APTT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ag</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Act</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RIPA</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;Heiti SC&quot;;</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>方法</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>指标</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBox_method">
      <item>
       <property name="text">
        <string>T-test</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ANOVA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mann-Whitney</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kruskal-Wallis</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>p-value</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string> </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="creat.qrc"/>
 </resources>
 <connections/>
</ui>
